docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurement and
+recommendation is more complex and needs to incorporate various other aspects. These
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
+  have been started, but are not ready when the next batch of recommendations is requested.
+  Without further consideration, the algorithm would be likely to recommend the pending
+  measurements again, as it is unaware they were already started.
+- **Partial targets**: When dealing with multiple targets that require very different
+  amounts of time to measure, the targets of previously recommended points might only be
+  partially available when requesting the next batch of recommendations. Still, these
+  partial measurements should ideally be considered when generating the recommendations.
+
+
+## Pending Measurements
+
+With *pending measurements* we mean recommendations whose measurement process has
+been started, but not yet completed by time of triggering the next set of
+recommendations. This is not to be confused with a situation where the targets of a
+recommendation have been partially measured. A measurement is *pending* whenever one or
+more of its targets are not measured yet. Only once all the targets are available,
+the data point can be removed from `pending_experiments` and be provided via 
+`measurements`.
+
+To avoid repeated recommendations in the above scenario, BayBE provides the 
+`pending_experiments` keyword. It is available wherever recommendations can be
+requested, i.e. [`Campaign.recommend`](baybe.campaign.Campaign.recommend) or 
+[`RecommenderProtocol.recommend`](baybe.recommenders.base.RecommenderProtocol.recommend).
+
+```{admonition} Supported Acquisition Functions
+:class: important
+`pending_experiments` is only supported by Monte Carlo (MC) acquisition functions, i.e. the
+ones that start with a `q` in their name. Attempting to use a non-MC acquisition
+function with `pending_experiments` will result in an 
+[`IncompatibleAcquisitionFunctionError`](baybe.exceptions.IncompatibleAcquisitionFunctionError).
+```
+
+```{admonition} Supported Recommenders
+:class: important
+For technical reasons, not every recommender is able to fully or even partially
```

Comment:
I think the wording of `fully or even partially` is confusing since I would not know what "partially" should even mean in this context. Maybe rephrase into something like "Note that different recommenders handle pending points in different ways"?

Reply:
was changed